index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Assignment 2</title>
+    <script src="jquery-3.4.1.min.js"></script>
+<script src="script.js"></script>
+
+<!--
+		Donjeta Emerllahu
+		INFO2340
+		12/2019
+		-->
+</head>
+<body>
+    <h1>The Wizard of OZ!</h1>
+    <p>
+    <strong><span id="female_1_1"></span></strong> 
+    live in the midst of the great
+    <strong><span id="state_1_1"></span></strong>
+    <strong><span id="geo_1_1"></span></strong> with Uncle 
+    <strong><span id="male_1_1"></span></strong>Who was a 
+    <strong><span id="occupation_1_1"></span></strong> and 
+    Aunt
+    <strong><span id="female_2_1"></span></strong>
+     who was the farmer's wife. Their house was small, for the lumber to build it had to be carrdied by
+    wagon many miles. There were four walls, a floor and a roof, which made one room; and this room 
+    a floor and a roof, which made one room; and this room contained a rusty looking
+    <strong><span id="noun_1_1"></span></strong>, a 
+    <strong><span id="noun_2_1"></span></strong> for the dishes,
+    a table, three or four chairs, and the beds.  
+    Uncle<strong><span id ="male_1_2"></span></strong> and 
+    Aunt <strong><span id ="female_2_2"></span></strong> had a big bed in one corner, and 
```

Comment:
I see how you used numbering in the id to keep these straight. Makes sense to me!